### ISSUE TITLE
make new icons for the app

### ISSUE THREAD

**Issue by GoogleCodeExporter** [De7vID/klingon-assistant#26]
```
A better set of icons is needed for the app.

The icons should not contain any trademarked symbols (unless used with 
permission).

Specifications:
http://developer.android.com/guide/practices/ui_guidelines/icon_design_launcher.
html#size
Summary - 36x36 (low), 48x48 (medium), a 72x72 (high), and (optionally) 96x96 
(extra high). Also, a 512x512 image is needed for the Play Store (which may or 
may not be related to the app's icon).

Background information:
- original icon:
  http://klingon-assistant.googlecode.com/git-history/88d815ccc5524df7053c7bc9c78938ff7043630c/KlingonAssistant/res/drawable/ic_dictionary.png
- current icons:
  http://klingon-assistant.googlecode.com/git/KlingonAssistant/res/drawable-ldpi/ic_ka.png
  http://klingon-assistant.googlecode.com/git/KlingonAssistant/res/drawable-mdpi/ic_ka.png
  http://klingon-assistant.googlecode.com/git/KlingonAssistant/res/drawable-hdpi/ic_ka.png
  http://klingon-assistant.googlecode.com/git/KlingonAssistant/icons/ic_ka_512.png

```

Original issue reported on code.google.com by `De.vID.jonpIn` on 8 Jan 2013 at 3:31

**Comment by GoogleCodeExporter** [comment]
Original comment by `De.vID.jonpIn` on 9 Jan 2013 at 5:13
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Upping the priority on this. I really dislike the current icons and want to 
replace them ASAP. :-)
```

Original comment by `De.vID.jonpIn` on 11 Jan 2013 at 10:38
- Added labels: **Priority-High**
- Removed labels: **Priority-Low**

**Comment by GoogleCodeExporter** [comment]
```
Fixed as of revision a82801e66137. We can make minor changes if necessary, but 
as far as I'm concerned the new icons are great!
```

Original comment by `De.vID.jonpIn` on 10 Feb 2013 at 8:58
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fixed as of revision a82801e66137. We can make minor changes if necessary, but 
as far as I'm concerned the new icons are great!
```

Original comment by `De.vID.jonpIn` on 10 Feb 2013 at 8:58
